doc/nrf/includes/cmd_build_and_run.txt

```diff
@@ -0,0 +1,29 @@
+Complete the :ref:`command-line build setup <build_environment_cli>` before you start building |NCS| projects on the command line.
+
+To build and program the source code from the command line, complete the following steps:
+
+1. Open a terminal window.
+#. Go to the specific sample or application directory.
+   For example, the folder path is :file:`ncs/nrf/applications/asset_tracker_v2` when building the source code for the :ref:`asset_tracker_v2` application |cmd_folder_path|.
+
+#. Make sure that you have the required version of the |NCS| repository by pulling the |NCS| repository, `sdk-nrf`_ on GitHub using the procedures described in :ref:`dm-wf-get-ncs` and :ref:`dm-wf-update-ncs`.
+
+#. To get the rest of the dependencies, run the ``west update`` command as follows:
+
+   .. code-block:: console
+
+      west update
+
+#. To build the sample or application code, run the ``west build`` command as follows:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build -b *build_target* -d *destination_directory_name*
+
+   The parameter *build_target* must be |cmd_build_target|.
```

Comment:
The bootloader is something that is built automatically with the secure board type even when you build a non-secure application, and in theory it would be best to build the crypto samples for the non-secure board as well. Then the crypto operations happen in the secure TF-M image, and you have an extra layer of security in case someone find a vulnerability in the non-secure application. However, I think the crypto part of TF-M isn't ready yet for 2.0.0. So there might be some reasons to use the secure board for now. Can you please correct me if I'm wrong @joerchan . But for the nrf9160, the general recommendation is to let the modem do the encryption with certificates stored securely in the modem. So the crypto sample is only relevant in case someone needs to encrypt from the application.

Reply:
When using the modem then the non-secure board is required. Otherwise using the secure should be fine, and up to the user to decide based on their security requirements.
The crypto support in TF-M is still in experimental state, apart from the random number generation.

Reply:
Then we could keep nrf9160dk_nrf9160 build target for now.

Reply:
- Why would anyone use nrf9160 without the modem? It doesn't provide any other connectivity and the price would eliminate it against alternatives without cellular connectivity. So I'd argue that all our customers use the nrf9160 because of it's modem.
- [The introduction to this user guide states](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7986/nrf/ug_nrf91_features.html#application): «The user application runs in the non-secure domain. Therefore, it must be built for the nrf9160dk_nrf9160_ns or thingy91_nrf9160_ns build target.»
- According to our [Zephyr docs](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/services/tfm/overview.html#non-secure-processing-environment): «At present, Zephyr can not be configured to be used as the secure processing environment.»

Please remove the secure board here, as it's only going to confuse users. If we think anyone want to write their own secure domain application, we should write a separate guide for that. For now I think it is enough that the secure-board is documented in the board page.